force-app/main/educationCloudSettings/lwc/educationCloudSettings/educationCloudSettings.html

```diff
@@ -1,15 +1,20 @@
 <template>
     <div class="slds-theme_default eda-height_full">
-        <ul class="slds-grid">
-            <li class="slds-col slds-size_4-of-12">
+        <ul class="slds-grid slds-wrap">
```

Comment:
I'm hitting an overflow issue here on screen resize and on the mobile view. The cards exceed the page's white background.
![Screen Shot 2021-09-08 at 11 18 52 AM](https://user-images.githubusercontent.com/45603724/132548320-529d6f8f-2476-449c-8e3a-39c3026990bf.png)

Reply:
This issue is corrected in [PR-1867](https://github.com/SalesforceFoundation/EDA/pull/1867) which is a branch off of this branch.

Reply:
@MatthewBlanski This fix should now be merged into this branch.